frontend/tests/e2e/tests/eligibility/CoverageStatusCheckTest.js

```diff
@@ -9,6 +9,7 @@ const ERROR_MESSAGE = 'Please correct errors before submitting'
 const PHN_REQUIRED_MESSAGE = 'PHN is required'
 const DOB_REQUIRED_MESSAGE = 'Date Of Birth is required'
 const INVALID_DOB_ERROR_MESSAGE = 'Date of Birth must not be in the future'
+const MINIMUM_DATE_VALIDATION = 'Date must be later than 19000101'
```

Comment:
Can we just reuse constants from src/util/.constants.js? Not sure if the tests folder will have problems accessing that source though.

Reply:
Added constant for minimum date validation in constant.js. Its being used only for testcafe right now.. Other message constants can be reused. We can cover seperately.